Community/alumni.md

```diff
@@ -107,6 +125,10 @@ Machine Learning for Anomaly Detection in Open Source Communities
 
 Open-source software development is a collaborative effort that requires decentralized decision making from different developers and maintainers. In order to measure the progress of the project, It is important to quantify the code changes across time. CHAOSS provides analytics and metrics to help open source communities measure the impact of the developer’s work on the project and the impact of the project on the community. Augur is a prototyped implementation of the CHAOSS Project on open source software metrics which systematically integrates data from several open-source repositories, issue trackers, mailing lists, etc. Anamoly detection is a common data science strategy of finding extreme data points (outliers), whose features differ vastly from other normal data points. From an open-source software development perspective, it detects unusual surges and drops in development activities like code-commits, pull-requests, etc. This project aims to identify the different types of anomalies that are available from trace data and deliver a personalized notification to the user using several machine learning techniques.
 
+#### Links
+
+* [Project Link on GSoC Website](https://summerofcode.withgoogle.com/archive/2020/projects/5310055743750144/)
+* [Github](https://github.com/sarit-adh)
 

```

Comment:
blog posts: https://medium.com/@sarit.adhikari

Reply:
@vchrombie I have added al the changes review them and if any change is required let me know

Reply:
@pankaj892
I'm checking the changes, you can resolve the comments if you finished the work.